### ISSUE TITLE
Unable to Install Plugin

### ISSUE THREAD

**Issue by jkattampilly-te** [future-architect/eslint-plugin-vue-scoped-css#186]
Hello,

I am facing an error when trying to install this plugin using the npm command mentioned in the Readme. Error seems to be on a peer dependency. Not sure if this is related to dependencies in the plugin, or my project, or both. 

I am using node 14 and eslint 6.7. Appreciate any guidance on this. Thank you!

```
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: postcss@8.2.7
npm WARN node_modules/eslint-plugin-vue-scoped-css/node_modules/postcss
npm WARN   postcss@"^8.2.4" from eslint-plugin-vue-scoped-css@2.1.0
npm WARN   node_modules/eslint-plugin-vue-scoped-css
npm WARN     dev eslint-plugin-vue-scoped-css@"*" from the root project
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer postcss@"^8.3.3" from postcss-safe-parser@6.0.0
npm WARN node_modules/eslint-plugin-vue-scoped-css/node_modules/postcss-safe-parser
npm WARN   postcss-safe-parser@"^6.0.0" from eslint-plugin-vue-scoped-css@2.1.0
npm WARN   node_modules/eslint-plugin-vue-scoped-css
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: postcss@8.2.7
npm WARN node_modules/eslint-plugin-vue-scoped-css/node_modules/postcss
npm WARN   postcss@"^8.2.4" from eslint-plugin-vue-scoped-css@2.1.0
npm WARN   node_modules/eslint-plugin-vue-scoped-css
npm WARN     dev eslint-plugin-vue-scoped-css@"*" from the root project
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer postcss@"^8.3.3" from postcss-scss@4.0.3
npm WARN node_modules/eslint-plugin-vue-scoped-css/node_modules/postcss-scss
npm WARN   postcss-scss@"^4.0.0" from eslint-plugin-vue-scoped-css@2.1.0
npm WARN   node_modules/eslint-plugin-vue-scoped-css
npm ERR! code ETARGET
npm ERR! notarget No matching version found for eslint-utils@^3.0.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/username/.npm/_logs/2022-02-02T19_21_53_038Z-debug.log
```

**Comment by ota-meshi** [comment]
I can't reproduce your problem, but I expect it to be due to a slightly older version of the dependent postcss.
I will update and release the dependency postcss to 8.3.3.

**Comment by ota-meshi** [comment]
I can't reproduce the problem. Would you please share your repository?

**Comment by jkattampilly-te** [comment]
I can't share as its a private repo in company.. But I realized that the npm version of eslint-plugin-vue-scoped-css that was getting downloaded the second time i tried installing was still the old version instead of the new version that you updated. This was happening because we have a local assets registry that caches packages. So I haven't been able to test with the lastest version of eslint-plugin-vue-scoped-css yet. Let me resolve these issues and will update you once I have retried installing. 

Thank you!
